.travis.yml

```diff
@@ -1,12 +1,14 @@
 # Test
-sudo: required
 language: c++
 compiler: gcc
 env:
     - BUILDENV=arch BUILDTYPE=cmake-ninja
     - BUILDENV=ubuntu-14.04 BUILDTYPE=cmake-make
 services:
     - docker
+before_install:
+    - python3 -m pip install --user flake8
```

Comment:
flake8 should be installed using the package manager in the test platform docker files.